### ISSUE TITLE
Create host pages for all hosts

### ISSUE THREAD

**Issue by mattstratton** [arresteddevops/ado-hugo#346]
See https://github.com/mattstratton/castanet/blob/master/REFERENCE.md#host-pages for reference

**Comment by mattstratton** [comment]
This is what a guest `yml` file looks like:

`mstratton3.yml`

```
Name: mstratton3
"FullName": "Matt Stratton"
"Bio": "Matt Stratton is a DevOps Evangelist at [PagerDuty](https://www.pagerduty.com), where he help dev and ops teams advance the practice of their craft and become more operationally mature. He collaborates with PagerDuty customers and industry thought leaders in the broader devops community, and his license plate actually says \"DevOps\".  
<br /><br />
Matt has over 20 years experience in IT operations, ranging from large financial institutions such as JPMorganChase and internet firms, including Apartments.com. He has given presentations at ITSM focused events, ChefConf, DevOpsDays, Interop, PINK, and various local groups within the Chicagoland area."
"Twitter": "mattstratton"
"Site": "http://www.mattstratton.com/"
"GitHub": "mattstratton"
"LinkedIn": "https://www.linkedin.com/in/mattstratton"
"Facebook": "https://www.facebook.com/matt.stratton"
```

This needs to become a file in `content/guest/mstratton3.md` that looks like this:

```
+++
title = "Matt Stratton"
Thumbnail = "img/guests/mstratton3.png"
Linkedin = "mattstratton"
GitHub = "mattstratton"
Facebook = "matt.stratton"
date = "2017-04-27T13:53:32-05:00"
Twitter = "mattstratton"
Type = "guest"
Website = "https://www.mattstratton.com"
+++
Matt Stratton is a DevOps Evangelist at [PagerDuty](https://www.pagerduty.com), where he help dev and ops teams advance the practice of their craft and become more operationally mature. He collaborates with PagerDuty customers and industry thought leaders in the broader devops community, and his license plate actually says \"DevOps\".  
<br /><br />
Matt has over 20 years experience in IT operations, ranging from large financial institutions such as JPMorganChase and internet firms, including Apartments.com. He has given presentations at ITSM focused events, ChefConf, DevOpsDays, Interop, PINK, and various local groups within the Chicagoland area.
```

Note that there may be other fields in the guest file - tl;dr they should just be converted to a frontmatter field with the same name. Also, we don't need the escaped quotes in the content, but it's okay if they are there and we can clean up manually later. 

I don't THINK there are fields in the guest file that aren't in the example; I am pretty sure this is a comprehensive one.

**Comment by mattstratton** [comment]
Fixed in b51e627
